src/qt/bitcoingui.cpp

```diff
@@ -748,6 +748,19 @@ void BitcoinGUI::setNetworkActive(bool networkActive)
     updateNetworkState();
 }
 
+bool BitcoinGUI::updateHeadersSyncProgressLabel()
```

Comment:
The boolean indicates whether the label was updated or not. This reduces redundant calls in `bitcoingui.cpp`. I'll document it a bit more.